### ISSUE TITLE
Split `registry.rs` module by its concerns

### ISSUE THREAD

**Issue by NunoAlexandre** [radicle-dev/radicle-registry#258]
It's getting cumbersome and difficult to go through. We should think of how to better split it into separate modules based on different concerns.

**Comment by xla** [comment]
For clarification are you referring to `runtime/registry.rs`?

**Comment by NunoAlexandre** [comment]
Yes! The only `registry.rs` we have in this project. Will adjust the title anyway.
